proxy/backend/src/main/java/org/apache/shardingsphere/proxy/backend/communication/ProxySQLExecutor.java

```diff
@@ -143,16 +134,40 @@ private boolean isPostgreSQLOrOpenGaussStatement(final SQLStatement sqlStatement
      * @throws SQLException SQL exception
      */
     public List<ExecuteResult> execute(final ExecutionContext executionContext) throws SQLException {
+        return needAutoTransaction(executionContext) ? doExecuteWithinTransaction(executionContext) : doExecute(executionContext);
```

Comment:
Please rename needAutoTransaction to `isNeedImplictCommit` and rename doExecuteWithinTransaction to `doExecuteWithImplictCommit`.

Reply:
Refer to mysql document——https://dev.mysql.com/doc/refman/8.0/en/implicit-commit.html, implicit commit should be a proper noun.

Reply:
In addition, can we investigate whether other databases also have implicit commit behavior?

Reply:
`implicit commit` is a proper noun.I will rename methods to` isNeedImplicitCommitTransaction` and `doExecuteWithImplicitCommitTransaction`.